nexus-writer/src/nexus/hdf5_file/run_file_components/timeseries_file.rs

```diff
@@ -0,0 +1,294 @@
+use anyhow::{anyhow, bail, Result};
+use chrono::{DateTime, Utc};
+use hdf5::{
+    types::{FloatSize, IntSize, TypeDescriptor},
+    Dataset, DatasetBuilderEmpty, Group, H5Type,
+};
+use ndarray::{s, Dim, SliceInfo, SliceInfoElem};
+use std::fmt::Debug;
```

Comment:
```suggestion
```

Reply:
It's needed here as it's explicitly used in line 20.
```rust
pub(super) trait TimeSeriesDataSource<'a> : Debug {
```